_conferences/devfest-south-africa-2022.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "GDG DevFest"
+website: https://gdg.community.dev/events/details/google-gdg-cape-town-presents-devfest-south-africa-2022/
+location: Cape Town, South Africa
+
+date_start: 2022-11-25
+date_end:   2022-11-26
```

Comment:
Is it a two day event? On the website I only see "Friday, Nov 25 9:00 AM - 5:00 PM (SAST)".

If it's a single day event, please use the same date for `date_start` and `date_end`.

Reply:
Ah, gotcha. Updated.